modules/twinklebatchundelete.js

```diff
@@ -70,7 +72,20 @@ Twinkle.batchundelete.callback = function twinklebatchundeleteCallback() {
 			$pages.each(function(index, page) {
 				var $page = $(page);
 				var title = $page.attr('title');
-				list.push({ label: title, value: title, checked: true });
+				var $editprot = $page.find('pr[type="create"][level="sysop"]');
+				var isProtected = $editprot.length > 0;
+
+				var metadata = [];
```

Comment:
The array is not needed as there's just one piece of metadata, and we're unlikely to add more in the future.

Reply:
I did it to be parallel with `batchdelete`, although I do agree we're unlikely to add more.  I don't think size or user is likely to be desired.